MapboxNavigation/RouteTableViewController.swift

```diff
@@ -56,6 +56,8 @@ class RouteTableViewController: UIViewController {
             return
         }
         
+        guard routeProgress.congestionTravelTimesSegmentsByStep.indices.contains(routeProgress.legIndex),
+            routeProgress.congestionTravelTimesSegmentsByStep[routeProgress.legIndex].indices.contains(routeProgress.currentLegProgress.stepIndex) else { return }
```

Comment:
`indices.contains` is a very roundabout way of checking whether an index is within bounds. How about `routeProgress.legIndex < congestionTravelTimesSegmentsByStep.count`?